test.sh

```diff
@@ -344,6 +344,19 @@ curl --url "https://$SINGULARITY_HOSTNAME/login" \
   | tee test/clear_login_success \
   | grep "msg = user authenticated by password"
 
+# Force log out user
+orbit/warpdrive.sh -u user -d
+
+NEW_LOGIN_COOKIE="$(sed -nE 's:.*<td>Token</td>[ \t]*<td>([^<]*)</td>.*:\1:p' < test/clear_login_success | tr -d '\n')"
```

Comment:
yeah I think we definitely need a better way of passing the cookie, some html parsing is ok but I think this may be taking it too far lmao

Reply:
updated to use curl cookie saving